src/header.rs

```diff
@@ -287,12 +287,17 @@ impl<W: io::Write> UncompressedHeader for BitWriter<W, BigEndian> {
     self.write_bit(fi.sequence.still_picture)?; // still_picture
     self.write_bit(fi.sequence.reduced_still_picture_hdr)?; // reduced_still_picture_header
 
+    assert!(fi.sequence.level[0][0] <= 3);
+    assert!(fi.sequence.level[0][1] <= 7);
```

Comment:
Won't this fail because the default is [3, 9]?

Reply:
yeah my bad I added those asserts last minute

Reply:
removed